TABTestKit/Classes/Contexts/NavigationContext.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  NavigationContext.swift
+//  TABTestKit
+//
+//  Created by Kane Cheshire on 09/09/2019.
+//
+
+import Foundation
+
+public protocol NavigationContext {}
+public extension NavigationContext {
+	
+	func see<ScreenType: Screen>(_ screen: ScreenType) {
+		screen.await()
+	}
+	
+	func complete<ScreenType: Completable & Screen>(_ screens: ScreenType...) {
```

Comment:
Will add example tests on how to use this `...`:

`complete(screenA, screenB, screenC)`